pixart.js

```diff
@@ -1 +1,46 @@
+// Previous commit has commented-out code from first 6 steps, this is just the simplified code w/ comments
 
+$(document).ready(function() {
+
+	var click = 0;										// click counter (how many times you submitted a new color)
+	var colorInput;
+
+	$("div.controls").append("<div class='brush'></div><div class='brush'></div>"); // adds 2 more color swatches
+	$("div.brush").css("display", "inline-block");		// displays color swatches next to each other
+	$("div.brush").eq(1).css("margin-right", "4px"); 	// The first brush always had a white space between it and the next one???? I don't know why
```

Comment:
This is dangerously close to writing inline styles. Managing this spacing issue in the CSS would be more in line with best practices, and might actually give you some more control over figuring out the root of the weird spacing...